.github/workflows/tests.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       - run: ./tests/pytype_test.py --print-stderr
 
   mypy:
-    name: Run mypy against typeshed
+    name: Test the stubs with mypy
```

Comment:
ignorable nit: I prefer "Type check the stubs with mypy" (keeping with my comment about all the use of the word "test")

Reply:
Hmm, this one I will ignore, since we still run mypy against the `test_cases` directory in `mypy_test.py`, and that feels more like a "test" than a "type check" :)